editor/store/index.js

```diff
@@ -11,14 +11,18 @@ import reducer from './reducer';
 import { withRehydratation, loadAndPersist } from './persist';
 import enhanceWithBrowserSize from './browser';
 import store from './store';
+import { getEditedPostTitle } from './selectors';
 
 /**
  * Module Constants
  */
 const STORAGE_KEY = `GUTENBERG_PREFERENCES_${ window.userSettings.uid }`;
+const MODULE_KEY = 'core/editor';
 
-registerReducer( 'core/editor', withRehydratation( reducer, 'preferences' ) );
+registerReducer( MODULE_KEY, withRehydratation( reducer, 'preferences' ) );
 loadAndPersist( store, 'preferences', STORAGE_KEY, PREFERENCES_DEFAULTS );
 enhanceWithBrowserSize( store );
 
+registerSelectors( MODULE_KEY, { getEditedPostTitle } );
```

Comment:
I assume we could use the following:

```js
import * as selectors from './selectors';

registerSelectors( MODULE_KEY, selectors );
```

We would have to try it.

Reply:
I don't think we want to expose all selectors. Some are internal only.